superchain/configs/sepolia/ink.toml

```diff
@@ -4,6 +4,7 @@ public_rpc = "https://rpc-gel-sepolia.inkonchain.com"
 sequencer_rpc = "https://rpc-gel-sepolia.inkonchain.com"
 explorer = "https://explorer-sepolia.inkonchain.com"
 superchain_level = 0
+superchain_time = 0 # Missing hardfork times are inherited from superchain.toml
```

Comment:
You can remove all other `<fork>_time` entries.

We may add all individual forks to chain configs in the future, but that's not decided yet, see https://github.com/ethereum-optimism/superchain-registry/issues/751